public/index.html

```diff
@@ -32,7 +32,7 @@ <h2 class="text-2xl text-green-50 italic animate__animated animate__zoomIn">Fron
         <main id="About" class="bg-zinc-900 text-pink-700 relative flex">
             <div class="w-1/4 p-40">
                 <div class="mt-1 mr-1">
-                    <a href="docs/adhithya-resume.pdf" target="_blank">
+                    <a href="docs/adhithya-resume.pdf" target="_blank" class="hover:transition-transform hover:duration-300 hover:transform hover:scale-110">
```

Comment:
Have you checked the scaling on hover?

it's not working. because the default `display` value of a tag is `inline` and I think transforms won't work on inline elements or something.

You can change it to `block`

Also, as I mentioned earlier the `duration-300` shouldn't not be prefixed by `hover:`